src/components/editor/InputLiteral.jsx

```diff
@@ -3,9 +3,8 @@
 import React, {Component} from 'react'
 import PropTypes from 'prop-types'
 import { connect } from 'react-redux'
+// import { getProperty } from '../../reducers/index'
 import { setItems, removeItem } from '../../actions/index'
-import PropertyRemark from './PropertyRemark'
-import RequiredSuperscript from './RequiredSuperscript'
```

Comment:
will these imports be used elsewhere, or just removed altogether?

Reply:
Used in the ResourceTemplateForm and will be in the OutlineHeader component.